File: Models/Competition/LeagueMatchModel.cs
using System;
using champi.Libs.Extensions;

namespace champi.Models.Competition
{
    public class LeagueMatchModel
    {
        public long Id { get; set; }
        public long FirstTeamId { get; set; }
        public string FirstTeamName { get; set; }
        public long SecondTeamId { get; set; }
        public string SecondTeamName { get; set; }
        public int? FirstTeamScore { get; set; }
        public int? SecondTeamScore { get; set; }
        public long? WinnerTeamId { get; set; }
        public string WinnerTeamName { get; set; }
        public DateTime? MatchDate { get; set; }
        public string MatchDatePersian => MatchDate.ConvertToPersianDate();
    }
}

File: ClientApp/src/app/models/competition/league-match.model.ts
export class LeagueMatchModel {
    id: number;
    firstTeamId: number;
    firstTeamName: string;
    secondTeamId: number;
    secondTeamName: string;
    firstTeamScore?: number;
    secondTeamScore?: number;
    winnerTeamId?: number;
    winnerTeamName: string;
    matchDate?: Date;
    matchDatePersian: string;
    editable: boolean;
}
